packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -136,6 +140,20 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
       </Row>
       {canMute && (
         <>
+          {enableVideoVolume && (
+            <Row>
+              <NumericInput
+                value={volume}
+                onChange={onChangeVolume}
+                min={0}
+                max={1.0}
+                isFloat
+                suffix={__('Volume', 'web-stories')}
+                aria-label={__('Volume', 'web-stories')}
+                data-testid="volume-input"
```

Comment:
Let's not use data-testid if we can target elements differently, e.g. by aria-label

Reply:
No we can't sadly. aria-label / labels do not work with numberic inputs.